src/linear_algebra/sesquilinear_form.lean

```diff
@@ -133,6 +133,63 @@ lemma map_sum_right {α : Type*} (S : sesq_form R M I) (t : finset α) (g : α 
   S w (∑ i in t, g i) = ∑ i in t, S w (g i) :=
 by haveI s_inst := is_add_monoid_hom_right S w; exact (finset.sum_hom t (λ z, S w z)).symm
 
+variables {M₂ : Type w} [add_comm_group M₂] [module R M₂]
+
+/-- Apply the linear maps `f` and `g` to the left and right arguments of the sesquilinear form. -/
+def comp (S : sesq_form R M I) (f g : M₂ →ₗ[R] M) : sesq_form R M₂ I :=
```

Comment:
Perhaps `comp_linear_maps` for consistency with `multilinear_map.comp_linear_map`?

Reply:
I was thinking about consistency with `linear_algebra.bilinear` forms where `comp` was used.

Reply:
That's probably a better comparison